docs/integration/docker-autodiscovery.md

```diff
@@ -8,46 +8,46 @@ Container orchestration tools like Docker Enterprise, Google Kubernetes or RedHa
 
-Sematext SPM-Client Auto-Discovery solves the issues above by the automatic discovery of all relevant information for application monitoring:
-- SPM-Client container connects to Docker Remote API and Kuberntes API to discover new application containers and network parameters
-- SPM-Client connects to container networks and Kubernetes pods for monitoring 
-- Application containers can be tagged with the SPM_TOKEN,  and SPM-Client uses the SPM_TOKEN label or environment variable to ship metrics to the right Monitoring App in Sematext Cloud (or Sematext Enterprise). 
+Sematext Agent Auto-Discovery solves the issues above by the automatic discovery of all relevant information for application monitoring:
+- Sematext Agent container connects to Docker Remote API and Kuberntes API to discover new application containers and network parameters
+- Sematext Agent connects to container networks and Kubernetes pods for monitoring
+- Application containers can be tagged with the SPM_TOKEN, and Sematext Agent uses the SPM_TOKEN label or environment variable to ship metrics to the right Monitoring App in Sematext Cloud (or Sematext Enterprise).
 
-## How does SPM Client Autodiscovery work? 
+## How does Sematext Agent Autodiscovery work?
 
-The SPM Client container gets distributed to every Kubernetes, Docker Swarm or Docker Enterprise node. Then it starts watching container events and compares the image name of started containers with a list of application images defined in a configuration template called `autoDiscovery.yml`. Then SPM-Client joins the container network and configures SPM-Client for the right IP-address and port and other settings according to the configuration template. 
+The Sematext Agent container gets distributed to every Kubernetes, Docker Swarm or Docker Enterprise node. Then it starts watching container events and compares the image name of started containers with a list of application images defined in a configuration template called `autoDiscovery.yml`. Then Sematext Agent joins the container network and configures Sematext Agent for the right IP-address and port and other settings according to the configuration template.
 
-Monitoring with Sematext Cloud requires the SPM_TOKEN for each application type. The SPM_TOKEN is read from the application container environment or container labels. 
+Monitoring with Sematext Cloud requires the SPM_TOKEN for each application type. The SPM_TOKEN is read from the application container environment or container labels.
```

Comment:
Is it SPM_TOKEN or something else?

Reply:
Replaced with MONITORING_TOKEN. Also for Node agent, checked with Stefan.